ts/test/session/unit/receiver/drop_incoming/shouldProcessMessage_test.ts

```diff
@@ -0,0 +1,130 @@
+import { describe } from 'mocha';
+import Sinon from 'sinon';
+import Long from 'long';
+import { expect } from 'chai';
+import { TestUtils } from '../../../../test-utils';
+import { shouldProcessContentMessage } from '../../../../../receiver/common';
+import { DURATION } from '../../../../../session/constants';
+
+describe('shouldProcessContentMessage', () => {
+  let envelopeTs: number;
+  beforeEach(() => {
+    TestUtils.stubWindowLog();
```

Comment:
I forgot but is it okay to commit stubWindowLog()? I thought it was just for testing purposes? 

Seems we have it committed in a few places.